src/Compiler/Checking/CheckExpressions.fs

```diff
@@ -4511,20 +4511,25 @@ and TcTupleType kindOpt (cenv: cenv) newOk checkConstraints occ env tpenv isStru
         else
             let argsR,tpenv = TcTypesAsTuple cenv newOk checkConstraints occ env tpenv args m
             TType_tuple(tupInfo, argsR), tpenv
+            
+and CheckAnonRecdTypeDuplicateFields (ident: _ -> Ident) elems = 
```

Comment:
I don't get the purpose behind the first argument (`ident`) here?

Reply:
This seems to be used as the identity function to get the correct text and range as we are iterating over the same elements to find duplicates. But I think we can remove it here :) 

I took this from https://github.com/dotnet/fsharp/blob/0dfd880da573fbed6ab1008c9b53cc9aacf39f67/src/Compiler/Checking/CheckDeclarations.fs#L390.